doc/source/Resources/User_Guide.rst

```diff
@@ -45,6 +45,184 @@ The same result can be obtained with the following code:
+        >>> hfss["$PrjVar3"] = "34 * $PrjVar1/$PrjVar2"
+        >>> hfss["$PrjVar3"]
+        2.13628300444106
+        >>> hfss["$PrjVar3"].value
+        2.13628300444106
+        hfss.variable_manager["$PrjVar3"].expression
+        '34 * $PrjVar1/$PrjVar2'
+
+
+.. image:: ./variables_advanced.png
+  :width: 600
+  :alt: Variable Management
+
+
+Modeler
+~~~~~~~
+Object-oriented programming is used to create and manage objects in the AEDT 3D and 2D Modelers. 
+You can create an object and change properties using getters and setters.
+
+.. code:: python
+
+    Create a box, assign variables, and assign materials.
+
+    from pyaedt.hfss import Hfss
+    with Hfss as hfss:
+         box = hfss.modeler.primitives.create_box([0, 0, 0], [10, "dim", 10],
+                                                  "mybox", "aluminum")
+         print(box.faces)
+         box.material_name = "copper"
+         box.color = "Red"
+
+
+
+.. image:: ./aedt_box.png
+  :width: 800
+  :alt: Modeler Object
+
+Once an object is created or is present in the design (from a loaded project), user can
+simply get the related object using getters. Getter works either with object id or object name.
+Object returned has all features even if it has not been created in PyAEDT.
+Next example shows how easily user can go deeper into edges and vertices of faces or 3d objects.
+
+.. code:: python
+
+
+     box = hfss.modeler.primitives["mybox2"]
+     for face in box.faces:
+        print(face.center)
+        for edge in face:
+            print(edge.midpoint)
+            for vertice in edge.vertices:
+                print(edge.position)
+     for vertice in box.vertices:
+        print(edge.position)
+
+
+All objects allows to execute any modeler operations like union or subtraction.
+
+
+.. code:: python
+
+
+     box = hfss.modeler.primitives["mybox2"]
+     cyl = hfss.modeler.primitives["mycyl"]
+     box.unit(cyl)
+
+
+.. image:: ./objects_operations.gif
+  :width: 800
+  :alt: Object Modeler Operations
+
+
+Mesh
+~~~~
+Mesh operations are very important in Engineering Simulation. PyAEDT is able to read all mesh
+operations already present in a design, edit them and create new ones.
+All mesh operations are listed into mesh object.
+
+.. code:: python
+
+
+    from pyaedt import Maxwell3d
+    m3d = Maxwell3d()
+    all_mesh_ops = m3d.mesh.meshoperations
+    my_mesh_op = all_mesh_ops[0]
+    # all properties are in props dictionary.
+    my_mesh_op.props["my_prop"] = "my_value"
+    my_mesh_op.update()
+
+
+.. image:: ./Mesh_Operations.png
+  :width: 800
+  :alt: Mesh object List
+
+Once an object is created or is present in the design (from a loaded project), user can
+simply get the related object using getters. Getter works either with object id or object name.
+Object returned has all features even if it has not been created in PyAEDT.
+Next example shows how easily user can go deeper into edges and vertices of faces or 3d objects.
```

Comment:
@maxcapodi78 I think this is a copy/paste from the example above for the faces, edges, vertices...
I think we can delete it.